docs/src/api.md

```diff
@@ -33,6 +33,10 @@ where `ϵ` is the step size of leapfrog integration.
   - Adapt the mass matrix `metric` of the Hamiltonian dynamics: `mma = MassMatrixAdaptor(metric)`
     
       + This is lowered to `UnitMassMatrix`, `WelfordVar` or `WelfordCov` based on the type of the mass matrix `metric`
+      + There is an experimental way to improve the *diagonal* mass matrix adaptation using gradient information (similar to [nutpie](https://github.com/pymc-devs/nutpie)), 
+      currently to be initialized for a `metric` of type `DiagEuclideanMetric`
+      via `mma = AdvancedHMC.Adaptation.NutpieVar(size(metric); var=copy(metric.M⁻¹))` 
```

Comment:
Is the idea to not export any new functionality in this PR? Do that later together with a change of interface?

Reply:
I thought so. But we can also export it - I've seen that AdvancedHMC _does_ export e.g. `WelfordVar` as well.

Reply:
I'm happy not to export and figure out the interface part later if you think that's better. I can see that it might be cleaner, having to do less changes to the interface if we collect them into a single bigger one. Just wanted to make sure I'm understanding correctly.

If we would export something new then we would need a version bump and a HISTORY.md entry explaining the change.